coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/query/repository/ContributorQueryRepository.java

```diff
@@ -32,7 +32,7 @@
   @Query(
       "MATCH (p)-[:CONTAINS_COMMIT]->(c:CommitEntity) WHERE ID(p) = {0} AND c.name = {1} WITH c LIMIT 1 "
           + "CALL apoc.path.subgraphNodes(c, {relationshipFilter:'IS_CHILD_OF>'}) YIELD node WITH node as c ORDER BY c.timestamp DESC WITH collect(c) as commits "
-          + "CALL apoc.cypher.run('UNWIND commits as c MATCH (c)<-[:CHANGED_IN]-(f) WHERE f.path = {path} RETURN f', {commits:commits, path: {2}}) YIELD value "
+          + "CALL apoc.cypher.run('UNWIND commits as c MATCH (c)<-[:CHANGED_IN]-(f) WHERE f.path = path RETURN f', {commits:commits, path: {2}}) YIELD value "
           + "WITH value.f as f, commits WITH commits, f "
           + "OPTIONAL MATCH (f)-[:RENAMED_FROM*0..]->(f2) WITH collect(f) + collect(f2) as files, commits "
```

Comment:
I believe you can change this query to be identical to the one for modules below (save for the `STARTS WITH`). This should make them easier to maintain in case of a change. We should also look into extracting common parts of queries somehow.

Reply:
Done